client/components/BusinessProfile/BusinessProfilePage.js

```diff
@@ -41,91 +42,51 @@ class BusinessProfilePage extends Component {
 	}
 
 	render() {
-		const { currentBusiness, id, reviews, userId, isLoading, allUsers } = this.props;
+		const { currentBusiness, id, reviews, userId, isLoading } = this.props;
 		const { errors } = this.state;
 
-		if (isLoading) { return <Spinner />; }
+		// if (isLoading) { return <Spinner />; }
```

Comment:
Hey jerry, nice work but I would advice that you clean up commented codes to keep your work more professional.

Reply:
Thanks for the feedback, I'll delete the commented codes.